_events/sr2021/league-2.md

```diff
@@ -1,10 +1,20 @@
 ---
 title: Competition League Session 2
-date: 2021-02-06
-time_tbc: true
+date: 2021-02-06 12:00:00 Europe/London
```

Comment:
oh, small thing: If we're committing to noon for all of these, should we update that too now?